### PR TITLE
Desktop version: Complete the desktop version of my portfolio website using media queries.

### PR DESCRIPTION
**Pull request by ClaudiaRojasSoto**
Complete the desktop version of my portfolio website using media queries, while following best practices for HTML/CSS and using a combination of Grid and Flexbox to lay out the elements on the page. The main goal is to replicate the design from the Figma template as much as possible and create an adaptable UI through the analysis of the Figma design.

Details:

- It uses media queries to ensure the website displays correctly on desktop screens.
- It uses a combination of Grid and Flexbox to style the page elements.
- Reproduces the Figma template design as closely as possible, including font, colors, images, text, and margins.
- Follow best practices for HTML/CSS and ensure no linter errors.
- Document work professionally.
- Implements a responsive UI through analysis of Figma's design.
- Use the correct GitHub flow.

I appreciate in advance your comments and suggestions to improve my work. I remain attentive to your observations. Thank you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -268,6 +344,7 @@ <h3>LET’S CONNECT</h3>
             </li>
         </ul>
     </section>
+
```
Comment (bobb-Rob) [inline index.html:347]: Your project is very beautiful and well-structured. But I see a few things that can be done to further improve on it. 

- [x] In the contact section, if you look at the Figma design, the background Illustrations are different for mobile and desktop. But in your project design, you still have the same `background-image: url('your-mobile-image-path')`. I suggest you use media queries to add the different illustrations according to the view. Please fix this. 

Figma design
image

File: index.html [hunk 2]
```diff
@@ -174,25 +230,44 @@ <h2 class="long-text">Multi-Post Stories</h2>
...
                 </li>
             </ul>
+            <div class="project-link-container">
             <a href="#" class="project-link">See project</a>
         </div>
+            </div>
+        </div>
     </section>
     <section class="about-me">
+        <div class="aboutme-desktop">
```
Comment (bobb-Rob) [inline index.html:270]: - [x] In the About Me section, there are two columns, the one on the left has the `About Me` text and the one on the right has the `languages-framework-skills`. If you look closely at the Figma design, you will see that the left column covers about 40% of the available space and the right column has 60%.  But in your design, it is almost 50-50 columns. In the image below labeled `your design` has a green `X` on it, that red circle shows the part that should be moved slightly to the left. Kindly fix this as you should stick to the design as much as possible. 

Figma Design
image

Your design
image

File: index.html [hunk 3]
```diff
@@ -174,25 +230,44 @@ <h2 class="long-text">Multi-Post Stories</h2>
...
+            <p class="subtitle-desktop-card4">
+                UBER
+                <span class="grey">
+                    <img src="./img/period.svg" alt="Grey period in text">
+                    Lead Developer
+                    <img src="./img/period.svg" alt="Grey period in text">
+                    2018
+                </span>
+            </p>
+            <p class="content-card4">
```
Comment (bobb-Rob) [inline index.html:242]: - [x] Your project cards look great and the button hover effect works just as expected, well done 👏🏾; But the card container also has a hover effect on it. In the Figma image below 👇🏾 , I have placed three labels `A`, `B`, and `C`. The `A` mark shows the cursor pointer hovering on the project card, and the `B` mark shows the blue border and shadow effect on it as a result of the mouse hover. Then `B` mark is the normal card when the mouse is NOT hovering on it. 

Kindly implement this.

Figma Design
image

File: index.html [hunk 4]
```diff
@@ -27,6 +27,13 @@ <h2>Claudia</h2>
             <div>
                 <div class="menu-toggle">&#9776;</div>
             </div>
+            <div class="nav-desktop">
+              <ul>
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
```
Comment (bobb-Rob) [inline index.html:35]: - [ ] Also, please see the `background-image` of the headline section and ensure you use the correct illustration for the desktop from the Figma Design. Again, you can use media query for this. Kindly attach the correct illustration for the correct view.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @ClaudiaRojasSoto, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃
 
Meanwhile, you did very well 👏🏾 on this project because your mobile design is intact.



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @ClaudiaRojasSoto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file professional ✔️
## Optional suggestions
- [OPTIONAL] Kindly fix your MIT link, it goes to 404 page.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**